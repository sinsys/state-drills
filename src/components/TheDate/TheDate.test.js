import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import TheDate from './TheDate';

describe('TheDate Component', () => {
	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<TheDate />, div);
		ReactDOM.unmountComponentAtNode(div);
	})

	it.skip('ui renders as expected', () => {
		expect(
			renderer.create(<TheDate />).toJSON()
		).toMatchSnapshot()
	});
});