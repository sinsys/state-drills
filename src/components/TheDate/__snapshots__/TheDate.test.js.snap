// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDate Component ui renders as expected 1`] = `
<div>
  11/23/2019, 1:11:51 PM
</div>
`;
